sklearn/ensemble/_voting.py

```diff
@@ -223,8 +221,8 @@ def fit(self, X, y, sample_weight=None):
         return super().fit(X, transformed_y, sample_weight)
 
     def predict(self, X):
-        """ Predict class labels for X.
-
+        """Predict class labels for X.
+        
```

Comment:
Added spaces should be removed. This in particular makes flake8 CI fail..

Reply:
thanks for the feedback! 
I used pydocstyle for the numpy conventions, do you suggest I use flake8 to catch errors as well?

Reply:
Yes, better to use flake8 (that wrapps pycodestyle). I imagine pycodestyle would also error there. You need to run `git pull` before adding new commits to address this since I resolved conflicts in this branch.

Reply:
appreciate the advice!